android/src/main/java/io/wazo/callkeep/VoiceConnectionService.java

```diff
@@ -304,9 +307,18 @@ private void startForegroundService() {
         assert manager != null;
         manager.createNotificationChannel(chan);
 
+        Activity currentActivity = RNCallKeepModule.instance.getCurrentReactActivity();
+        Intent notificationIntent = new Intent(this, currentActivity.getClass());
```

Comment:
When I have to display a notification in background, my app crashes because of `java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference`. I think it happens because there is no currentActivity. There was a similar issue that was fixed with this PR: https://github.com/react-native-webrtc/react-native-callkeep/pull/576/files

Reply:
Yes my bad, it's fixed in https://github.com/react-native-webrtc/react-native-callkeep/pull/659, will be released today.

Reply:
Perfect! Thank you very much!!

By the way, great library.